content/Video_Display.md

```diff
@@ -725,7 +725,7 @@ Mode 2 - Searching OAM Period
 A typical procedure that waits for accessibility of VRAM would be:
 
 ```
-ld   hl,0FF41h    ;-STAT Register
+ld   hl,$FF41     ;-STAT Register
 @@wait:           ;
```

Comment:
This should be changed to `.wait` (and below), since whether `@@wait` is a valid RGBASM label is a bit complicated.

Reply:
Fixed

Reply:
Fixed